### PR TITLE
Format group comparison lollipop plot y-axis label

### PR DESCRIPTION
**Pull request by gblaih**
Fixes an issue in https://github.com/cBioPortal/cbioportal/issues/9901

### CODE REVIEW

File: src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx [hunk 1]
```diff
@@ -23,7 +23,21 @@ interface IGroupComparisonMutationsTabPlotProps {
 }
 
 function plotYAxisLabelFormatter(symbol?: string, groupName?: string) {
-    return `${symbol} ${groupName} Mutations`;
+    // lowercase = 1 and uppercase = 1.3 (based on 'w' and 'W'), if groupName >= 13, stop and + "..."
```
Comment (alisman) [inline src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx:26]: weird that the params of this function are optional    lets see what happens if we make them required

File: src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx [hunk 2]
```diff
@@ -23,7 +23,21 @@ interface IGroupComparisonMutationsTabPlotProps {
...
+    let length = 0;
+    let label = '';
+    for (let c of groupName!) {
```
Comment (alisman) [inline src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx:29]: why not take symbol into account too?

Comment (gblaih) [reply src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx:29]: the symbol is always % or # I believe with a space after so I just need to look at the groupName. I found that 13 'w' and 10 'W' with the symbol + space before it is the max.

File: src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx (hunk 1 above)
Comment (alisman) [reply src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx:26]: @gblaih

Comment (gblaih) [reply src/pages/groupComparison/GroupComparisonMutationsTabPlot.tsx:26]: Making them required doesn't seem to change anything so I will change it to that.

**Pull request merged**